content/shell/app/shell_main_delegate.cc

```diff
@@ -95,6 +95,12 @@
 #include "content/shell/app/ios/shell_application_ios.h"
 #endif
 
+#if BUILDFLAG(IS_IOS_TVOS)
```

Comment:
Is this code used in cobalt builds? Shouldn't we be using //cobalt/shell?

Reply:
Got it, makes sense. Though note at least as of now we wouldn't be able to upstream and //cobalt/shell changes, as //cobalt/shell doesn't exist upstream

Reply:
Sorry for the confusion: by "upstreaming" here I meant sending a PR to this repository with a change we've made in our internal repository while working on the tvOS port, not submitting the changes to the Chromium repository.